tests/phpunit/integration/Core/Authentication/Clients/OAuth_ClientTest.php

```diff
@@ -337,6 +337,7 @@ public function test_get_proxy_setup_url() {
 		$this->assertContains( 'nonce=', $url );
 		$this->assertContains( 'return_uri=', $url );
 		$this->assertContains( 'action_uri=', $url );
+		$this->assertContains( 'application_name=', $url );
```

Comment:
Can you add this to the assertion block below too? Since that parameter should also be included once the site/user is authenticated and there is a site ID.

Reply:
will do

Reply:
See https://github.com/google/site-kit-wp/pull/1572/commits/5d03cb73b0433c8244dd2dbfaba9e4dc1281ad3a